hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -159,6 +159,7 @@ mkSimpleTx (txin, TxOut owner valueIn datum refScript) (recipient, valueOut) sk
       { txIns = map (,BuildTxWith $ KeyWitness KeyWitnessForSpending) [txin]
       , txOuts = outs
       , txFee = TxFeeExplicit fee
+      , txValidityRange = (TxValidityLowerBound 0, TxValidityUpperBound 1)
```

Comment:
Why are we doing this? It will make transactions created by `mkSimpleTx` only validate on slot 0 and 1 (not sure if these are inclusive).

Reply:
We needed to provide validity ranges so the validator works. I tried using the network tip here but it seems that in this scenario onchain part always sees this tx to be valid starting at slot 0.

Reply:
This tx is actually not used in the on-chain validation so I'll revert this change.